pkg/kubelet/apis/config/v1beta1/defaults_test.go

```diff
@@ -1071,12 +1068,6 @@ func TestSetDefaultsKubeletConfiguration(t *testing.T) {
 
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
-			if len(tc.config.FeatureGates) > 0 {
-				for k, v := range tc.config.FeatureGates {
-					featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, featuregate.Feature(k), v)
```

Comment:
This hid this issue in unit tests, since it was setting the feature gates more betterer than we would really observe in the wild